samples/ingestion/ingestion-client/FetchTranscription/FetchTranscription.csproj

```diff
@@ -16,6 +16,7 @@
     <OutputPath>bin\Release\</OutputPath>
   </PropertyGroup>  
   <ItemGroup>
+    <PackageReference Include="Azure.AI.Language.Conversations" Version="1.1.0-beta.1" />
```

Comment:
can't we get all of the classes in Connector/Serializable/Conversations also from this package here as well?

Reply:
also, I don't think we should have a "-beta.1" version in here

Reply:
The Azure.AI.Language.Conversations SDK is auto-generated using the cadl and it does not return the classes.